Example/Sources/Random.swift

```diff
@@ -0,0 +1,356 @@
+import UIKit
+import Foundation
+
+// each type has its own random
+public extension Bool {
+    /// SwiftRandom extension
+    public static func random() -> Bool {
```

Comment:
We would have to include the LICENSE for this project and a link to it if we're going to use it to this extent